### ISSUE TITLE
custom block editor opens on deletion

### ISSUE THREAD

**Issue by yogo1212** [nomeata/incredible#132]
when i delete a custom block, an editor pops up

**Comment by nomeata** [comment]
Hmm, I tried to reproduce it, but couldn’t. Can you give step-for-step instructions, or post a screen capture video?

**Comment by michaelgwelch** [comment]
I've seen this but I don't think it's a custom block being discussed but an "annotation block"

https://user-images.githubusercontent.com/826112/183489467-80d445cf-4604-404d-b372-bcf8e2ebf66e.mov

**Comment by nomeata** [comment]
Thanks! That clarifies things.

As you can see in your recording, it even happens when moving the block.

It looks like the on-click-handling, which opens the dialogue, started to act even when something else happened (moving, deletion). I wonder if this is due to a recent in one of the used libraries.

It doesn’t seem to be too disastrous, so I’ll have a look when I am touching the code anyways. But if someone else wants to give it a shot: It’s plain JavaScript, and I guess the bug is somewhere here: https://github.com/nomeata/incredible/tree/master/webui

**Comment by michaelgwelch** [comment]
> As you can see in your recording, it even happens when moving the block.

Yeah, i mentioned that, but my comment was under the video and easily missed.

**Comment by yogo1212** [comment]
oops! i only played with it a bit and found the bug - sorry for the confusion!
and thanks, @michaelgwelch for adding the correction :-)
